charts/templates/storageclass_default.yaml

```diff
@@ -0,0 +1,14 @@
+{{- if .Values.createStorageClass }}
+apiVersion: storage.k8s.io/v1
+kind: StorageClass
+metadata:
+  annotations:
+    storageclass.kubernetes.io/is-default-class: "true"
+  name: encrypted-storage
```

Comment:
maybe we should add the name of the underlying storage class to the name?
so sth like "encrypted-pd". There seems to be at least on other storage system on GCP (Kubernetes File Store).

Reply:
I see the point, but I also see some problems with this.
For one, we don't ship storage classes with different disk types (Users can create those themselves, if desired), so we only add some extra information to the name of the storage class that is also available when looking at the storage class spec.
Another thing is that if we add the disk type to the storage class name, deployments need (small) adjustments when moved from e.g. Azure to GCP. 
Right now both CSPs provide similar named storage classes.

As a reference the storage classes created by default on GKE using the gce-pd driver are `standard-rwo` and `premium-rwo`